extensions/hooks/_hook_copy_pdbs_to_package.py

```diff
@@ -18,23 +19,32 @@ def post_package(conanfile):
     # Find dumpbin path
     output = StringIO()
     try:
+        program_files = get_env("ProgramFiles(x86)") or get_env("ProgramFiles")
```

Comment:
I'd say we can use `os.environ.get` in this case, rather than the legacy `get_env`

Or alternatively the string for the command can use `%VAR%` vs `$VAR` conditional on winbash, if we can evaluate it.

I'd also add a comment that `get_env("ProgramFiles")` is a fallback for 32-bit windows

Reply:
I had some issues with variable expansion and the special characters in bash, and after a few tries I ipted for this approach. Will use `os.environ.get`, thanks!